sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1783,7 +1783,7 @@ async def test_queue_receive_keep_conn_alive_async(self, servicebus_namespace_co
                 assert len(messages) == 0  # make sure messages are removed from the queue
                 assert receiver_handler == receiver._handler  # make sure no reconnection happened
 
-    @pytest.mark.skip(reason="TODO: iterator support")
+    @pytest.mark.skip(reason="TODO: _counter doesnt exist in pyamqp")
```

Comment:
What were we counting?

Reply:
(I believe the handler._counter.get_current_ms() was uamqp's equivalent to time.time() in pyamqp.) I'll unskip/refactor this in the switch pr.

Reply:
We should be able to update the test in this PR - it's just swapping the method call for time.time()